src/pick/__init__.py

```diff
@@ -1,4 +1,5 @@
 import curses
+from textwrap import fill
```

Comment:
This import statement is not sorted alphabetically. And since `fill` is a very common word, I'd like to use it with its module name, for example `textwrap.fill(...)`.

Apart from that, this change looks good to me